frontend/components/about/ValuesItems.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+
+    import type { values } from '.';
+    const props = defineProps<values>();
+
+</script>
+
+
+<template>
+    <li class="ValuesItem__list">
+        <img class="ValuesItem__image" :src="imageUrl" alt="image">
```

Comment:
Линтеру не нравится оформление компонента с несколькими пропсами в одну строку.
Для `alt` нужно либо использовать перевод (когда картинка несёт в себе смысл), либо просто не указывать его.
https://htmlacademy.ru/blog/html/alt-text  
```suggestion
        <img 
                class="ValuesItem__image" 
                :src="imageUrl" 
        >
```